src/structured_config.py

```diff
@@ -152,6 +155,27 @@ def sqlalchemy_max_overflow_validator(cls, value: str) -> Optional[int]:
             return int_value
         raise ValueError("Value out of range.")
 
+    @validator("global_async_queries_polling_delay")
+    @classmethod
+    def global_async_queries_polling_delay_validator(
+        cls, value: str
+    ) -> Optional[int]:
+        """Check validity of `global_async_query_polling_delay` field.
+
+        Args:
+            value: global-async-query-polling-delay value
+
+        Returns:
+            int_value: integer for global-async-query-polling-delay configuration
+
+        Raises:
+            ValueError: in the case when the value is out of range
+        """
+        int_value = int(value)
+        if 500 <= int_value <= 5000:
```

Comment:
We're better off with literals (min and max values here) being defined somewhere that shows what they mean and can be tweaked if needed.

Reply:
I have followed this pattern for all of the values in `structured_config` - I'll make a separate PR which creates literals with definitions for all of them.